packages/data-point/lib/reducer-types/resolve.test.js

```diff
@@ -38,9 +38,12 @@ describe('reducer#resolve', () => {
     })
 
     const reducer = { type: 'INVALID TYPE' }
-    expect(() => {
-      Resolve.resolve(manager, accumulator, reducer)
-    }).toThrowErrorMatchingSnapshot()
+    return Resolve.resolve(manager, accumulator, reducer)
+      .catch(e => e)
+      .then(result => {
+        expect(result).toBeInstanceOf(Error)
+        expect(result).toMatchSnapshot()
```

Comment:
Maybe we want to use `toThrowErrorMatchingSnapshot()` here?

Reply:
Just for record keeping, we agreed not to do this because it's using a promise instead of a simple function.